request/admin.py

```diff
@@ -39,7 +39,7 @@ def request_from(self, obj):
             user = obj.get_user()
             return format_html(
                 '<a href="?user__username={0}" title="{1}">{2}</a>',
-                user.username,
+                user.get_username(),
```

Comment:
I think it will be better to use a primary key (`user.pk`) in the filter instead of `get_username()`, i.e.:
```python
return '<a href="?user={0}" title="{1}">{2}</a>'.format(
    user.pk,
    _('Show only requests from this user.'),
    user,
)
```
I haven't noticed that before, sorry.